service/src/intTest/java/uk/nhs/adaptors/gp2gp/gpc/GetGpcDocumentComponentTest.java

```diff
@@ -37,6 +37,15 @@
 @DirtiesContext(classMode = DirtiesContext.ClassMode.AFTER_EACH_TEST_METHOD)
 public class GetGpcDocumentComponentTest extends BaseTaskTest {
     private static final String DOCUMENT_NAME = EhrStatusConstants.DOCUMENT_ID + ".json";
+    private static final String DOCUMENT_URL = "https://orange.testlab.nhs.uk/B82617/STU3/1/gpconnect/documents/fhir/Binary/";
+    private static final String EXPECTED_ERROR_RESPONSE = "The following error occurred during Gpc Request: "
+        + "{\n  \"resourceType\": \"OperationOutcome\",\n  \"meta\": {\n    "
+        + "\"profile\": [ \"https://fhir.nhs.uk/StructureDefinition/gpconnect-operationoutcome-1\" ]\n  },\n  "
+        + "\"issue\": [ {\n    \"severity\": \"error\",\n    "
+        + "\"code\": \"invalid\",\n    \"details\": {\n      \"coding\": [ {\n        "
+        + "\"system\": \"https://fhir.nhs.uk/ValueSet/gpconnect-error-or-warning-code-1\",\n        "
+        + "\"code\": \"NO_RECORD_FOUND\",\n        \"display\": \"No Record Found\"\n      } ]\n    },\n    "
+        + "\"diagnostics\": \"No record found\"\n  } ]\n}";
```

Comment:
Remove this expected error response, it isn't used

Reply:
removed